src/OrchardCore.Modules/OrchardCore.Templates/Views/Template/Create - Copy.cshtml

```diff
@@ -0,0 +1,143 @@
+@model TemplateViewModel
```

Comment:
Fix the file name of this file

Reply:
thanx, i kept it as a reference until the PR was ready